apps/ejabberd/src/ejabberd_router.erl

```diff
@@ -326,9 +424,10 @@ compile_routing_module() ->
 
 default_routing_modules() ->
```

Comment:
I think my question about the default order got lost in prev PR. Is it possible to have `mongoose_router_localdomain` before all external component routers by default? Routing to local user is the most frequent case.